components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -311,7 +313,8 @@ func (r *Reconciler) syncConditionSubscriptionActive(subscription *eventingv1alp
 	condition := eventingv1alpha1.MakeCondition(eventingv1alpha1.ConditionSubscriptionActive, eventingv1alpha1.ConditionReasonSubscriptionActive, corev1.ConditionTrue, "")
 	if !isActive {
 		logger.Debugw("Waiting for subscription to be active", "name", subscription.Name, "status", subscription.Status.EmsSubscriptionStatus.SubscriptionStatus)
-		condition = eventingv1alpha1.MakeCondition(eventingv1alpha1.ConditionSubscriptionActive, eventingv1alpha1.ConditionReasonSubscriptionNotActive, corev1.ConditionFalse, "")
+		message := fmt.Sprintf("current subscription status: %s", subscription.Status.EmsSubscriptionStatus.SubscriptionStatus)
```

Comment:
(nit) Whats the benefit of this message? Won't the user see value `subscription.Status.EmsSubscriptionStatus.SubscriptionStatus` in the status? Can we write something like: "Waiting for subscription to be active.."